lib/fluent/plugin/out_http.rb

```diff
@@ -0,0 +1,213 @@
+require 'net/http'
+require 'uri'
+require 'openssl'
+require 'fluent/plugin/output'
+require 'fluent/plugin_helper/socket'
```

Comment:
unnecessary require.

Reply:
This is needed because out_http uses several constants from this module.

Reply:
sorry… I missed it…